docs/content/en/docs/installation/configuration.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Configure Tetragon"
+linkTitle: "Configuration"
+weight: 5
+---
+
+## Configuration
```

Comment:
you don't need to repeat the title
<img width="669" alt="image" src="https://github.com/cilium/tetragon/assets/11256051/4a46f3b7-28fc-460d-a4fd-2e1735138b31">